### DISCUSSION TITLE
Why don't we register GDPR webhooks

### DISCUSSION THREAD

**Discussion by geneowak** [Kyon147/laravel-shopify#319] [Q&A]
According to the [wiki](https://github.com/Kyon147/laravel-shopify/wiki/Creating-Webhooks#creating-a-webhook-job) the GDPR webhooks shouldn't be registered in the config file. 
Is there a particular reason why? 
What happens if someone registers them?

**Comment by Kyon147** [comment]
Here is the wiki on GDPR hooks https://github.com/Kyon147/laravel-shopify/wiki/GDPR
